app/http.php

```diff
@@ -643,6 +641,10 @@ function removeAllRuntimes(Table $activeRuntimes, Orchestration $orchestration):
 
             $activeRuntimes->del($runtimeName);
 
+            $error = empty($output)
```

Comment:
This was previously at the top on catch, because above, on line ~624, we override it. Logic should be:

1. Get errors from exception
2. Try to get logs from logs.txt. If successful, use that instead